### PR TITLE
Drop support for Django 1.6

### LINKED ISSUE #108: Drop support for Django 1.6
Update the build and docs.

### PR DESCRIPTION
**Pull request by damonkelley**
Closes #108.

Review the updated version of the docs [here](http://django-name.readthedocs.org/en/drop-1.6-support/) The only pages that have changed are `installation.rst` and `development.rst`

### CODE REVIEW

**Comment by damonkelley** [comment]
ping @ldko @somexpert

**Comment by somexpert** [comment]
:thumbsup:

File: docs/installation.rst [hunk 1]
```diff
@@ -77,45 +77,7 @@ Installation
 5. Migrate/Sync the database.
```
Comment (ldko) [inline docs/installation.rst]: With no 1.6 this could be "Migrate the database" and we could keep the "$ ./manage.py migrate name" line.

Comment (damonkelley) [inline docs/installation.rst]: Good catch!

Comment (damonkelley) [inline docs/installation.rst]: Fixed @ldko

**Comment by ldko** [comment]
:+1:

**Pull request merged**